core/test/element_geometric_operations_test.cpp

```diff
@@ -46,3 +46,20 @@ TYPED_TEST(ElementPositionNTest, unzip_position) {
   EXPECT_EQ(geometry::detail::component<T::value>::overloads(m), m);
   EXPECT_THROW(geometry::z(K), except::UnitError);
 }
+
+TEST(ElementRotationTest, rotate_vector) {
+  Eigen::Vector3d vec(1, 2, 3);
+  Eigen::Quaterniond rot(4, 5, 6, 7);
```

Comment:
Using `AngleAxisd` to initialise the Quaternions might need to easier to understand test that are more robust. For example, `AngleAxisd(M_PI / 2, Vector3f::UnitY())` applied to `Vector3d(1, 0, 0)` -> `Vector3d(0, 0, 1)`.

Reply:
great suggestion